src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -125,6 +125,46 @@
     /** Whether the trip must be wheelchair accessible. */
     public boolean wheelchairAccessible = false;
 
+    /**
+     * When in wheelchair-accessible mode we add some penalties when traversing places where we don't
+     * have accessibility information or we know to be wheelchair inaccessible.
+     *
+     * This is the cost of traversing a stop without accessibility information.
+     */
+    public int unknownWheelchairAccessAtStopPenalty = 600;
+    /**
+     * The cost of traversing a stop which is wheelchair-inaccessible.
+     */
+    public int noWheelchairAccessAtStopPenalty = 3 * unknownWheelchairAccessAtStopPenalty;
+    /**
+     * The cost of boarding a trip without accessibility information.
+     */
+    public int unknownWheelchairAccessAtTripPenalty = 1200;
+    /**
+     * The cost of boarding a trip which we know to be wheelchair-inaccessible.
+     */
+    public int noWheelchairAccessAtTripPenalty = 3 * unknownWheelchairAccessAtTripPenalty;
+    /**
+     * The reluctance of using stairs in a wheelchair. This value will be multiplied with the
+     * regular cost of traversing the stairs which depends on its length.
+     *
+     * In other words 100 meter long stairs are 10 times worse than a 10 meter one.
```

Comment:
This comment doesn't quite make sense. A 100 meter long stairs are already 10x longer than a 10 meter long stairs. Suggested wording as follows:

```suggestion
     * In other words, a 100 meter long stairs would now seems like a 2,000 meter long stairs after applying this reluctance factor.
```